docs/src/code/style-guide.adoc

```diff
@@ -252,16 +282,126 @@ of .c and .h are reserved for plain C. Headers are for class, method,
 and structure declarations, not code (unless the functions are declared
 inline).
 
-== Python coding standards
+=== Python coding standards
 
 Use the http://www.python.org/dev/peps/pep-0008/[PEP 8] style for
 Python code.
 
-== Comp coding standards
+=== Comp coding standards
 
 In the declaration portion of a .comp file, begin each declaration at
 the first column. Insert extra blank lines when they help group related
 items.
 
 In the code portion of a .comp file, follow normal C coding style.
 
+== Code describing the functionality for the human
+
+This is a very recent (1/2022) part of this document. Please help shaping it if you are familiar with asciidoc.
+
+After the website, and maybe the one or other YouTube Video, the
+LinuxCNC documentation is likely the first point of contact for any new
+user. The talent to get the documentation right is mostly disjunct from
+computationally orchestrating all the moving parts of a mill or lathe,
+though. Still, we need this to shine if we want LinuxCNC to shine and
+for a transfer of our knowledge for the next generation - this seems
+fair to say for a project that was started in the last millenium and few
+individuals buying their own mills/lathes before their hair turns gray.
+
+=== Overall structure of documentation
+
+There are two basic documents, i.e. the
+ * Users' Guide and the
+ * Developers' Guide
+
+All documents created belong to either of these "parental documents"
+and are included from the respective document, either directly or that
+included file includes it.
+
+=== Common challenges
+
+ * header tags block like toc etc
+   
+   Every file should start with a header. This is typically
+   ----
+   :lang: en
+   ----
```

Comment:
Shouldn't code blocks be surrounded by blanck lines ?

Reply:
Let's decide: Yes. :)